File: homework/src/main/java/models/Publication.java
package models;

import lombok.Builder;
import lombok.Getter;

import java.util.Date;

@Getter
@Builder
public class Publication {
    private Integer stationId;
    private String city;
    private Integer temp;
    private Double rain;
    private Integer wind;
    private String direction;
    private Date data;

    @Override
    public String toString() {
        return "{" +
                "(stationId," + stationId +
                "),(city,\"" + city + '\"' +
                "),(temp," + temp +
                "),(rain," + rain +
                "),(wind," + wind +
                "),(direction,\"" + direction + '\"' +
                "),(data," + data +
                ")}";
    }
}

File: homework/models/Publication.py
class Publication:
    def __init__(self, stationId, city, temp, rain, wind, direction, data):
        self.stationId = stationId
        self.city = city
        self.temp = temp
        self.rain = rain
        self.wind = wind
        self.direction = direction
        self.data = data

    def __str__(self):
        return "{" + \
                "(stationId," + str(self.stationId) + \
                "),(city,\"" + self.city + '\"' + \
                "),(temp," + str(self.temp) + \
                "),(rain," + str(self.rain) + \
                "),(wind," + str(self.wind) + \
                "),(direction,\"" + str(self.direction) + '\"' + \
                "),(data," + str(self.data) + \
                ")}"
